resources/boilerplates/flattenData.js

```diff
@@ -15,10 +16,11 @@ export function flattenChildData(element) {
 export function flattenData(element) {
   const flatData = {
     ...element.instance,
+    children: []
   };
 
-  if (element.children) {
-    flatData.children = element.children.map(
+  if (element.children && element.children.length > 0) {
+    flatData.children = element.children.filter(child => !!child).map(
```

Comment:
Is it really necessary still for checking whether the length is greater than 0?  And to filter out the nulls?  We should be doing that...